packages/axelar-wasm-std/src/address.rs

```diff
@@ -288,4 +294,30 @@ mod tests {
             address::Error::InvalidAddress(..)
         );
     }
+
+    #[test]
+    fn validate_stacks_address() {
+        // account
+        let addr = "ST319CF5WV77KYR1H3GT0GZ7B8Q4AQPY42ETP1VPF";
+        assert!(address::validate_address(addr, &address::AddressFormat::Stacks).is_ok());
```

Comment:
```suggestion
        assert_ok!(address::validate_address(addr, &address::AddressFormat::Stacks));
```
you can use the `assert_ok` or `assert_err_contains` like in our tests that are a bit better

Reply:
Done.